great-idea-website/index.html

```diff
@@ -16,53 +16,67 @@
   <section class="top-content">
-    Features
-    Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
-  
-  
-    About
-    Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+    <div class="left">
+      <h3>Features</h3>
+      Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+    </div>
+    <div class="right">
+      <h3>About</h3>
+      Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+    </div>
   </section>
  
   <img class="middle-img" src="img/mid-page-accent.jpg" alt="Image of code snippets across the screen">
   
   <section class="bottom-content">
-      Services
-      Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+      <div>
+        <h3>Services</h3>
+        Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+      </div>
     
-    
-      Product
-      Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
-    
-      Vision
-      Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
-  </section>
+      <div class="product">
+        <h3>Product</h3>
+        Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+      </div>
+      <div>
+        <h3>Vision</h3>
+        Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+      </div>
+    </section>
+
+    <hr />
+
 
   <section class="contact">
-      Contact
-      123 Way 456 Street
-      Somewhere, USA
-      1 (888) 888-8888
+      <h3>Contact</h3>
+      123 Way 456 Street<br />
+      Somewhere, USA<br /><br />
+      1 (888) 888-8888<br /><br />
       sales@greatidea.io
   </section>
 
```

Comment:
Overall very nice work for your first day, just a couple of small things to add. 👍